x-pack/plugins/ml/public/application/components/ml_inference/components/test_pipeline.tsx

```diff
@@ -134,6 +200,19 @@ export const TestPipeline: FC<Props> = memo(({ state, sourceIndex }) => {
         gutterSize="xs"
         data-test-subj="mlTrainedModelsInferenceTestStep"
       >
+        {sourceIndexMissingError && showCallOut ? (
+          <EuiFlexItem grow={false}>
+            <EuiCallOut
```

Comment:
I think this callout should be closer to the panel for entering the sample document. How's it look directly above the row of buttons, or above 'View request body'?